x-pack/plugins/observability_solution/observability_onboarding/public/application/quickstart_flows/auto_detect/get_auto_detect_command.tsx

```diff
@@ -20,6 +20,7 @@ export function getAutoDetectCommand(
       --install-key=${options.installApiKey}
       --ingest-key=${options.ingestApiKey}
       --ea-version=${options.elasticAgentVersion}
+      --integration-config-file=${options.confDownloadUrl}
```

Comment:
We don't need a new parameter for this endpoint. We already have the `--kibana-url` param so we can use that to construct the URL to the integrations config ourselves the same way we're already doing for all the other Kibana endpoints the auto-detect script is using.